assets/terraform/gce/bootstrap/centos.sh

```diff
@@ -134,7 +136,6 @@ done
 
 # Make changes permanent
 cat > /etc/sysctl.d/50-telekube.conf <<EOF
-fs.may_detach_mounts=1
```

Comment:
Is this also not required for centos/rhel 7?

Reply:
You're correct, this does default to 0 on centos/rhel 7.  I made it a conditional in https://github.com/gravitational/robotest/pull/268/commits/14aacd83dd04aa99e6e8b3dc19f0c7f48a3a70a9#diff-d70484b0d10572ea985f7b4c534ea1f4R145-R149